hw2-turtle-graphics/turtle-stretch1.js

```diff
@@ -0,0 +1,149 @@
+//Codecore Week 2 homework
```

Comment:
feel free to include stretch and non-stretch exercises together in one file so that you only need to submit one file with all the logic.